web/src/emoji_picker.ts

```diff
@@ -286,14 +310,14 @@ function toggle_reaction(emoji_name, event) {
         return;
     }
 
-    const message_id = current_message_id;
+    const message_id = current_message_id ?? 0;
```

Comment:
Is it safe to default the message_id to 0. If we don't have the `message_id` we should not `toggle_reaction` in the first place IMO.